setup.py

```diff
@@ -32,9 +32,9 @@
     include_package_data=True,
     zip_safe=True,
     install_requires=[
+        'importlib-resources >= 1.3;python_version<"3.9"',
```

Comment:
This backport needs to be imported differently: the module name is `importlib_resources`, not `importlib.resources`.

However, given that Python 3.8 EOL date was in October 2024, I think you can remove this backport completely and bump required Python to 3.9.